restapi/stories.go

```diff
@@ -190,9 +190,29 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+	names := r.Form["tags"]
+	if len(names) != 0 {
+		err = api.AddTags(names, story)
+		if err != nil {
+			return rest.ErrInvalidRequest(api.logger, "Error Adding Tags", err)
+		}
+	}
 	return rest.MsgStatusOK("Story Added Successfully")
 }
 
+func (api api) AddTags(names []string, story models.Story) error {
+	var length = len(names)
+	tags := make([]models.Tag, length)
```

Comment:
no need to declare the var if we are only using it one place 
```suggestion
	tags := make([]models.Tag, len(names))
```